docs/make.jl

```diff
@@ -1,7 +1,15 @@
-using Documenter, Example
+using Documenter, Weave, Example
 
-makedocs(modules = [Example], sitename = "Example.jl")
+for file ∈ readdir(joinpath(dirname(pathof(Econometrics)), "..", "docs", "jmd"))
```

Comment:
This feels like it would only confuse people. Like you need to do some `weave` stuff to run the documentation. Also weave is a bit of a heavy / complicated dependency.